README.md

```diff
@@ -511,6 +514,55 @@ name:      lcd_1602_i2c
 web site:  https://github.com/raspberrypi/pico-examples/tree/HEAD/i2c/lcd_1602_i2c
 ```
 
+## verify
+
+This command allows you to verify the contents of Flash/SRAM on the device matches a file, similar to the `--verify` option for `save` and `load`.
```

Comment:
Presumably it verifies the contents of Flash by default, and to get it to verify the contents of SRAM you'd need to pass it custom `<from>` and `<to>` addresses?